examples/stupid-s.mkd

```diff
@@ -113,6 +112,15 @@ Finally, where STuPiD becomes part of a client standard
 protocol as hinted in {{xmpp}}), it is a good
 thing if STuPiD is already defined.
 
+A summary table explains the above:
+
+|Reason| Summary|
+|---------------|
+|diff-comm-int| There are different communities of interest|
+|common-code| Having common code will optimize implementations|
+|incorporation| a standard is easy to incorporate in other standards|
```

Comment:
```suggestion
|Reason|Value|Summary|
|:-----|----:|:------|
|diff-comm-int| 1 | There are different communities of interest|
|common-code| 2 | Having common code will optimize implementations|
|incorporation| 3 | a standard is easy to incorporate in other standards|
{: #ex-table title="Example Table"}
```

Reply:
Of course, I'd not use the cumbersome markdown syntax, but cols="l r l".

(Oh, and we still need to invent colspan/rowspan syntax.
Was thinking about <<<< for cells filled by colspan and ^^^^ for rowspan.
The of course the whole v3 table generation needs to be done.)